Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Localization/LocalizationFilesTest.cs

```diff
@@ -13,14 +13,14 @@ namespace Aquality.WinAppDriver.Tests.Localization
 {
     public class LocalizationFilesTest
```

Comment:
should not these tests be in the core?

Reply:
should be here and there also, to check current localization files. will copy them to core. 
Issue in core: https://github.com/aquality-automation/aquality-selenium-core-dotnet/issues/24